docs/sources/operations/storage/_index.md

```diff
@@ -109,6 +110,14 @@ Resources: `*`
 Resources: `arn:aws:iam::<aws_account_id>:role/<role_name>`
 
 
+### IBM Cloud Object Storage
+
+When using IBM Cloud Object Storage (COS) as object storage, the following permissions are needed:
```

Comment:
since it is one permission, let's rephrase this statement:
>When using IBM Cloud Object Storage (COS) as object storage, IAM `Writer` role is sufficient.